sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -1193,7 +1193,7 @@ def test_sincos_rewrite_sqrt():
     for p in [1, 3, 5, 17]:
         for t in [1, 8]:
             n = t*p
-            for i in range(1, (n + 1)//2 + 1):
+            for i in range(1, min((n + 1)//2 + 1, 10)):
```

Comment:
It is the product `n` that is used, the biggest value is 136. Obviously the loops could have been combined but that does not save time. Those numbers are chosen so that cos and sin of `pi/n` can be expressed in terms of sqrt or, equivalently, are constructible by compass and straightedge. See https://en.wikipedia.org/wiki/Constructible_polygon and https://en.wikipedia.org/wiki/Compass-and-straightedge_construction . (Luckily, 257 was not tested.)